packages/console/src/contexts/SubscriptionDataProvider/use-subscription-data.ts

```diff
@@ -1,26 +1,106 @@
-import { condString } from '@silverhand/essentials';
-import { useContext } from 'react';
+import { pick } from '@silverhand/essentials';
+import { useContext, useEffect, useMemo } from 'react';
+import useSWR from 'swr';
 
-import { defaultTenantResponse } from '@/consts';
+import { useCloudApi } from '@/cloud/hooks/use-cloud-api';
+import { type LogtoSkuResponse, type NewSubscriptionUsageResponse } from '@/cloud/types/router';
```

Comment:
can further rename `NewSubscriptionUsageResponse` to `SubscriptionUsageResponse` in this change.